fn main() {}

fn v() -> isize { //~ ERROR mismatched types
mod _ { //~ ERROR expected identifier
pub fn g() -> isizee { //~ ERROR cannot find type `isizee` in this scope
mod _ { //~ ERROR expected identifier
pub    g() -> is //~ ERROR missing `fn` for function definition
(), w20); //~ ERROR mismatched closing delimiter
}
(), w20); //~ ERROR mismatched closing delimiter
//~^ ERROR expected item
}
